src/screens/HomeScreen.tsx

```diff
@@ -49,7 +49,7 @@ export const HomeScreen: FC = () => {
         backdropComponent={renderBackdrop}
       >
         <View style={[s.p4]}>
-          <Text>Awesome 🎉</Text>
+          <Text style={[s.textPrimary]}>Awesome 🎉</Text>
```

Comment:
To się schowa `automatycznie` jak tylko będziemy mieli wyciągnięty `Text` do osobnego komponentu

Reply:
Myślę, że jak to już się wszędzie pojawia to można zacząć kombinować z wyciąganiem tego komponentu